communication/communications.py

```diff
@@ -77,8 +78,30 @@ def _write(self, sig, opcode, arg, seqNo):
         Sends the code to the Arduino
         """
         checksum = self.create_checksum(arg, opcode)
-        opcode_string = "%d%d%03d%d\r" % (sig, opcode, arg, checksum)
+        opcode_string = "%d%d%03d%d%d\r" % (sig, opcode, arg, checksum, seqNo)
         self.to_robot(opcode_string)
+        
+        # Check twice if response received, if not - resend
+        success = False
+        while (!success):
+            for i in range(2):
+                sleep(0.1)
+                response = self.internal_queue.peek()
+                if response:
+                    # Check if success
+                    # (not checksum fail or unrecognized or bad command length)
+                    # Note: would be better to have a specific response upon success
```

Comment:
Not really (they're all over the place and we would need to keep track if we add more).  It would be eadiest to just use one code for success